easybuild/easyconfigs/g/GLib/GLib-2.34.3-goolf-1.5.14.eb

```diff
@@ -15,6 +15,7 @@ sources = ['glib-%(version)s.tar.xz']
 dependencies = [
     ('libffi', '3.0.13'),
     ('gettext', '0.18.2'),
+    ('libxml2', '2.9.1', '', ('GCC', '4.8.2')),
```

Comment:
@geimer: PR updated, please merge and retest https://github.com/hpcugent/easybuild-easyconfigs/pull/1661